### ISSUE TITLE
key payment fails if max key fee is disabled

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2051]
## The Issue
If max key fee is set to null, you get an ugly error on paid content. It should allow the payment to go through. 

```
C:\Users\thoma\Desktop>lbrynet get porno
{
  "error": "'NoneType' object is not subscriptable"
}
```
```
2019-04-24 11:06:30,706 WARNING  lbrynet.extras.daemon.Daemon:852: Error downloading porno: 'NoneType' object is not subscriptable
```

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by tzarebczan** [comment]
@seanyesmunt just ran into this. Please fix, it should be simple. If anyone has their max purchase price set to no limit, paid downloads will fail.

**Comment by seanyesmunt** [comment]
This is a pretty bad bug

**Comment by tzarebczan** [comment]
Fixed in #2467
